KarhooUISDK/CommonUI/UIConstants.swift

```diff
@@ -189,6 +189,16 @@ public struct UIConstants {
 
             /// 95%
             public static let mainActionButtonPressedAffineTransform = CGAffineTransform(scaleX: 0.95, y: 0.95)
+
+            static let addressViewDotsColumnWidth: CGFloat = 10
+
+            static let addressViewLabelHeight: CGFloat = 32
+
+            static let addressViewPadding: CGFloat = 10
```

Comment:
Can we use any pre-defined spacing instead? Maybe try it out with 8 or 12 and see how it looks.  Same for the rest of the new constants except for scale factor.

Reply:
I did check how it looks and the current version (which looks like the one on design) is the best.